src/EpisodeIntegrationService/ProcessData/ProcessData.cs

```diff
@@ -110,17 +109,22 @@ private async Task ProcessEpisodeDataAsync(List<Episode> episodes, string episod
                 var modifiedEpisode = new
                 {
                     EpisodeId = episode.episode_id,
-                    episode.episode_type,
-                    episode.bso_organisation_code,
-                    episode.bso_batch_id,
-                    episode.episode_date,
-                    episode.end_code,
-                    episode.date_of_foa,
-                    episode.date_of_as,
-                    episode.appointment_made,
-                    episode.call_recall_status_authorised_by,
-                    episode.early_recall_date,
-                    episode.end_code_last_updated
+                    ParticipantId = episode.participant_id,
```

Comment:
As mentioned in the previous comment, we won't have all the values so ParticipantId, for example, can be left as null for now

Reply:
Updated properties to include only those from BS Select